tests/models/test_hooks.py

```diff
@@ -348,8 +348,6 @@ def on_test_model_train(self):
 
     expected = [
         'on_fit_start',
-        'on_pretrain_routine_start',
-        'on_pretrain_routine_end',
```

Comment:
why are these removed?

Reply:
I believe it's meant to be called during training only.
https://github.com/PyTorchLightning/pytorch-lightning/blob/4c6f36e6e14a5e3bace1fe32505ae0fe6f8bc682/pytorch_lightning/core/hooks.py#L87-L96

Reply:
You are right, but was the test failing before? :thinking:

Reply:
no, because these hooks were called during testing and the tests were expecting these hooks to be called.

Reply:
but now they aren't